website/routes.py

```diff
@@ -324,6 +332,16 @@ def make_url_map(app):
             json_renderer,
         ),
 
+        Rule(
+            [
+                '/project/<pid>/citation/<style>/',
+                '/project/<pid>/node/<nid>/citation/<style>/',
```

Comment:
Ah, that is true. Brian, thoughts?

On Wed, Feb 4, 2015 at 1:43 PM, Lyndsy Simon <notifications@github.com>
wrote:

> In website/routes.py
> <https://github.com/CenterForOpenScience/osf.io/pull/1710#discussion_r24107442>
> :
>
> > @@ -324,6 +332,16 @@ def make_url_map(app):
> >              json_renderer,
> >          ),
> >
> > +        Rule(
> > +            [
> > +                '/project/<pid>/citation/<style>/',
> > +                '/project/<pid>/node/<nid>/citation/<style>/',
>
> A node only has one citation - which is basically metadata. I considered
> /project/<pid>/citation/?style=<style> as well, but decided this was
> better since we make limited use of URL parameters elsewhere.
>
> In the near future, citations (plural) linked to a node might take the <node
> path>/citations/ namespace for other things, too.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/CenterForOpenScience/osf.io/pull/1710/files#r24107442>
> .
>